src/groups/mqb/mqbs/mqbs_filebackedstorage.cpp

```diff
@@ -148,7 +145,12 @@ FileBackedStorage::FileBackedStorage(
     // and domain instance will return a zero capacity meter when queries to be
     // passed to the 'FileBackedStorage' instance.
 
-    d_virtualStorageCatalog.setDefaultRda(config.maxDeliveryAttempts());
+    d_virtualStorageCatalog.setDefaultRda(
+        domain->config().maxDeliveryAttempts());
+
+    d_queueStats_sp.createInplace(d_allocator_p, d_allocator_p);
+    d_queueStats_sp->initialize(queueUri, domain);
+    d_queueStats_sp->setQueueContentRaw(0, 0);
```

Comment:
The stats must initialize to `0`, why `setQueueContentRaw`?
Maybe, we remove `setQueueContentRaw` method?

Reply:
We also call this method from `InMemoryStorage::setQueue()`. Though in this case the storage is just created, hence the queue is always empty. So yes, we can remove this method